cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -529,19 +539,6 @@ def make_federal_judge(item, testing=False):
         else:
             city, state, org = '', '', ''
 
-        position = Position(
```

Comment:
It seems weird to remove this. Doesn't removing this mean that `position` on line 547 isn't defined? Maybe I'm missing it?

Reply:
You're correct.  I'd gotten ahead of myself testing something else out and inadvertently deleted this when I shouldn't have.